lib/decanter/core.rb

```diff
@@ -57,11 +57,23 @@ def decant(args)
         return handle_empty_args if args.blank?
         return empty_required_input_error unless required_input_keys_present?(args)
         args = args.to_unsafe_h.with_indifferent_access if args.class.name == 'ActionController::Parameters'
-        {}.merge( unhandled_keys(args) )
+        {}.merge(default_values)
```

Comment:
The naming here is a little confusing to me. I don't thing `handled_keys` is great because it's a key-value hash, not an array of keys, but for consistency should this be `default_keys`?